sigle/src/styles/index.css

```diff
@@ -1 +1,8 @@
 @tailwind base;
+
+html,
+body,
+#root,
+#__next {
+  height: 100%;
```

Comment:
Context: after trying other solutions to implement centering the content below the header, this one allowed me to do so. 

Not sure if there is a desired place for global styles like this, but I'm aware of the`globalCss` function from stitches, so happy to put them there if preferred.

Image from suggested change:
![Capture d’écran 2022-01-18 à 13 12 42](https://user-images.githubusercontent.com/5749437/149935982-4ed569fc-6bfa-4c8f-b1c6-01eae890ada1.png)